mantidproject/mantid
# Add GUI to partially save a Mantid project

This adds a new GUI window to let the user choose what workspaces and associated window to save to a project file.

**To test:**
Clicking Save Project/Save Project As should work similarly to before, but you should now be shown an interface that allows you to select which workspaces you want to save as part of the project.

Please be ensure to check edge cases such as:
 - Windows belonging to multiple workspaces
 - Workspace groups
 - Complicated interfaces: slice viewer, VSI, spectrum viewer...

This should also be tested on multiple platforms to ensure consistency

Fixes #17969 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
